docs/automl_overview.md

```diff
@@ -214,15 +214,15 @@ Komer, Brent, James Bergstra, and Chris Eliasmith (2014).
 *ICML workshop on AutoML 2014*.
 
 ## ML-Plan
-[source](https://github.com/fmohr/AILibs/tree/master/softwareconfiguration/mlplan) |
-[documentation](https://github.com/fmohr/AILibs/tree/dev/softwareconfiguration/mlplan/docs/javadoc) |
+[source](https://github.com/starlibs/AILibs) |
+[documentation](https://starlibs.github.io/AILibs/projects/mlplan/) |
 Java |
-Optimization: Hierachical Task Planning | -
+Optimization: Best-First Search on a search graph induced through Hierachical Task Network Planning | -
```

Comment:
The '-' field should hold the license, the repo lists  AGPL-3.0. If that is correct, could you update that here?